### DISCUSSION TITLE
URI error when trying to login

### DISCUSSION THREAD

**Discussion by halfrubbish** [timdorr/tesla-api#749] [Q&A]
Just copied and pasted the example code but getting this error now. Any ideas what I've done wrong?

```

C:/Ruby32-x64/lib/ruby/3.2.0/uri/common.rb:724:in `URI': bad argument (expected URI object or URI string) (ArgumentError)
        from C:/Users/user1/.local/share/gem/ruby/3.2.0/gems/tesla_api-3.1.0/lib/tesla_api/client.rb:113:in `login!'
```

**Comment by neomilium** [comment]
Using login+password credentials, I'm facing the same error.
Digging the code, this error raises after a 403 (Forbidenn) HTTP status code. Using valid or invalid credentials, it fails the same.

I'm now able to use this gem using the access token method, not the login+password one.
